libs/did-resolver-lib/src/resolver.ts

```diff
@@ -10,16 +10,35 @@ type Purpose =
   | 'Service';
 
 export default class PeerDIDResolver implements DIDResolver {
+  private diddocs: DIDDoc[];
+
+  constructor(diddocs: DIDDoc[] = []) {
+    this.diddocs = diddocs;
+  }
```

Comment:
why do you have an array of DID documents here?

Reply:
i just merged the resolver in my branch to be able to use it i did not implement it